src/index.tsx

```diff
@@ -10,7 +10,18 @@ import ReactDOM from 'react-dom/client'
 import { createBrowserRouter, createRoutesFromElements, Route, RouterProvider } from 'react-router-dom'
 import { Alerts, AuthProvider, ErrorView, Layout } from './components'
 import './index.module.css'
-import { DatabasesList, DeploymentDetails, InstancesList, Login, NewDhis2Instance, SignUp, StackDetails, StacksList } from './views'
+import {
```

Comment:
Is this happening due to formatting done by your editor or due to running `yarn format`?

Reply:
Just checked and it's the IntelliJ IDEA that's making this change. Single line is preferred then I assume?

Reply:
I don't care much either way but unless there's a reason to change it... :man_shrugging: